content/images/icons/facebook.svg.licence

```diff
@@ -0,0 +1,2 @@
+Copyright 2022 Facebook
+See https://www.facebook.com/brand/resources/facebookapp/guidelines for brand guidelines
```

Comment:
Not sure why the license check is passing for this one -- it shouldn't, we need an SPDX license reference in here. I'll have a look.

Reply:
Ok, thanks. I discussed this with Peter, and he told me to set up a file with a link to the FB branding page. I looked for licences for all the social media icons and came up blank.

Reply:
Hm, yes there is a problem in the `dep5` file. It says:

```
Files: content/images/icons/* content/robots.txt content/favicon.ico
Copyright: 2020 seL4 Project a Series of LF Projects, LLC.
License: CC-BY-SA-4.0
```

Apparently we had only our own icons in there before. I'll try to make that list more precise so it doesn't declare additional icons as Creative Commons.

This still leaves us with the problem that the actual license for these new icons needs to be in the LICENSES file in the repository. A link is not sufficient.

Reply:
Ok, this should work:
```suggestion
Copyright 2022 Facebook
SPDX-License-Identifier: LicenseRef-UsedWithPermission

See https://www.facebook.com/brand/resources/facebookapp/guidelines for brand guidelines
```